src/v/cluster/metadata_cache.h

```diff
@@ -47,11 +47,11 @@ namespace cluster {
 /// |                                          |
 /// |         Metadata Cache (facade)          |
 /// |                                          |
-/// +----+-------------+----------------+------+
-///      |             |                |
-/// +----v----+   +----v-----+    +-----v------+
-/// | Members |   |  Topics  |    |  Leaders   |
-/// +---------+   +----------+    +------------+
+/// +---+----------+------------+----------+---+
```

Comment:
nit: while you're here could you also add a space at L32 (sorry, don't know how to get GH to let me comment on lines outside the diff)

Reply:
Also maybe remove the "metadata dissemination service." in the middle of the sentence? Not sure what it's there for

Reply:
Good call.